docs/en/observability/quickstarts/monitor-hosts-with-otel.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[quickstart-monitor-hosts-with-otel]]
+= Quickstart: Monitor hosts with OpenTelemetry
+
+preview::[]
+
+In this quickstart guide, you'll learn how to collect logs and metrics using the Elastic Distribution of OpenTelemetry (EDOT) Collector.
+You'll also learn how to get value out of your observability data after collecting it.
+
+[discrete]
+== Prerequisites
+
+* An {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data. This quickstart is available for all Elastic deployment models. To get started quickly, try out our hosted {ess} on {ess-trial}[{ecloud}].
```

Comment:
Any links to deployment models in case users are unaware?

Reply:
Let's recommend serverless, is now GA and much quicker! also let's be specific that is quicker for users that just want to try it out.
eg. The quickest way to try it out is creating a trial project in Elastic serverless.